library/src/main/java/com/novoda/downloadmanager/DownloadObserver.java

```diff
@@ -32,8 +29,8 @@ void release() {
         contentResolver.unregisterContentObserver(contentObserver);
     }
 
-    interface OnUpdate {
-        void onUpdate();
+    interface Callback {
```

Comment:
this is more of a `listener` than callback since multiple events will be triggered